core/domain/stats_domain.py

```diff
@@ -616,12 +690,27 @@ def __init__(self, answer_occurrences=None):
             answer_occurrences if answer_occurrences else [])
 
     def to_raw_type(self):
+        """Returns the answer occurrences list in its raw type.
```

Comment:
What exactly does "raw type" mean?

Reply:
"raw type" means it returns the value in the raw dict form- 
       ```return {
            'answer': self.answer,
            'frequency': self.frequency
        }```
Shall I remove this for the docstrings?

Reply:
`Returns the answer occurrences list with each answer represented as a Python dict.`

Then, in the "Returns" section, give details of the actual format of each item in the list.

Reply:
Done!